sensor/common/compliance/service.go

```diff
@@ -21,13 +22,15 @@ type Service interface {
 
 	Output() chan *compliance.ComplianceReturn
 	AuditEvents() chan *sensor.AuditEvents
+	NodeScans() <-chan *storage.NodeScanV2
```

Comment:
curious, do we ever send to the other channels? why is this the only receive-only channel?

Reply:
I am following a pattern where I explicitly identify writers and readers by using general channel in the first case and the read-only in the second case. Here, this helps us to discover early potential errors when the reader would try to write to a channel.